cookbooks/aws-parallelcluster-platform/templates/nvidia-imex/nvidia-imex.service.erb

```diff
@@ -0,0 +1,26 @@
+[Unit]
```

Comment:
If this file is taken directly from the nvidia-imex documentation, then link to the official doc documenting it. In this way if we face issues at some point with the service definition we have a quick reference to the suggested values from nvidia and also spot quickly the changes we made on top of it.